frontend/src/pages/servers/index.tsx

```diff
@@ -38,27 +62,60 @@ const getStats = (servers: Array<Essential>): Stats => ({
   servers: servers.length,
 });
 
-const Row = ({ s }: { s: Essential }) => (
-  <li className="hover-bg-black-10 lh-copy pa2 ph0-l bb b--black-10">
-    <Link href={"/servers/" + s.ip}>
-      <a className="link black flex items-center justify-between">
-        <div className="overflow-hidden">
-          <span className="db black-70 measure truncate">{s.hn}</span>
-          <span className="db black-30 f6 measure truncate">{s.gm}</span>
-        </div>
+const Row = ({ s }: { s: Essential }) => {
+  const { onCopy, hasCopied } = useClipboard(s.ip);
 
-        <div className="pr2 tr dn di-ns flex-shrink-0">
-          <div className="black-70">
-            {s.pa ? <span>🔐</span> : null} <span>{s.ip}</span>
-          </div>
-          <div className="db black-30 f6 measure">
-            {s.pc}/{s.pm} playing now
-          </div>
-        </div>
-      </a>
-    </Link>
-  </li>
-);
+  return (
+    <Box
```

Comment:
When you use <CardList /> as the parent for this list of rows, each child list item will receive a prop named `sx` with the standard list styles, you can pass this directly to the Box `sx` prop to get all the same border radius and stuff as the other card lists (thread list, post list, notification list, etc.)